apps/sms/test/marionette/lib/views/composer/accessors.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
```

Comment:
We've renamed ```composer``` to ```new_message``` recently, could you please rename as well? At least the part that is new in this patch.